tool/tsh/tsh.go

```diff
@@ -1034,31 +1034,45 @@ func onListClusters(cf *CLIConf) {
 		utils.FatalError(err)
 	}
 
-	var clusters []services.RemoteCluster
+	var rootCluster services.RemoteCluster
+	var leafClusters []services.RemoteCluster
 	err = client.RetryWithRelogin(cf.Context, tc, func() error {
 		proxyClient, err := tc.ConnectToProxy(cf.Context)
 		if err != nil {
 			return err
 		}
 		defer proxyClient.Close()
 
-		clusters, err = proxyClient.GetAllClusters(cf.Context)
-		return err
+		var rootErr, leafErr error
+		rootCluster, rootErr = proxyClient.GetRootCluster()
+		leafClusters, leafErr = proxyClient.GetLeafClusters(cf.Context)
+		return trace.NewAggregate(rootErr, leafErr)
 	})
 	if err != nil {
 		utils.FatalError(err)
 	}
+
+	profile, _, err := client.Status("", cf.Proxy)
+	if err != nil {
+		utils.FatalError(err)
+	}
+	showActive := func(cluster services.RemoteCluster) string {
```

Comment:
I would check with @benarent how we want to display things that are active in the CLI so we're consistent. I think for profiles for example, the active one is marked with a `> ` symbol in front of it. I'm doing similar thing with database access.

Reply:
We don't make this very obvious in our docs, so better suggestions may be welcomed :)

Reply:
Kept the column but renamed it to 'Selected' as recommended by @benarent in #4965.